client/blocks/login/index.jsx

```diff
@@ -140,11 +143,29 @@ class Login extends Component {
 					clientTitle: oauth2ClientData.title
 				}
 			} );
+			if ( oauth2ClientData.name && oauth2ClientData.name === 'woo' ) {
```

Comment:
we don't need to have this safeguard here, `if ( oauth2ClientData.name === 'woo' ) {` should be enough.